presto-spi/src/main/java/com/facebook/presto/spi/function/WindowFunctionOptions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.function;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Retention(RUNTIME)
+@Target(TYPE)
+public @interface WindowFunctionOptions
```

Comment:
I think if you implemented `ignore nulls` for three remaining `ValueWindowFunctions` (`FirstValue`, `LastValue` and `NthValue`), then we'd not need this annotation but instead we could distinguish `canIgnoreNulls`-functions by interface they implement (`canIgnoreNulls = instanceof ValueWindowFunctions`).

Reply:
I will do this.